x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleType.java

```diff
@@ -78,7 +78,7 @@ public class TimeseriesLifecycleType implements LifecycleType {
         AllocateAction.NAME,
         MigrateAction.NAME,
         FreezeAction.NAME,
-        RollupV2.isEnabled() ? RollupILMAction.NAME : null
+        RollupILMAction.NAME
```

Comment:
The rollup action will be available but it will not operate on indices that are not time series indices. 

So, in a similar way as we did for `TimeSeriesIdFieldMapper` I did not put this behind a feature flag. I can add it though, if you think it makes sense.

Reply:
Fixed in [86c3347](https://github.com/elastic/elasticsearch/pull/85708/commits/86c3347410aaf35140a79d839ff0494d1edcae35)